src/components/Form/Collection/Collection.jsx

```diff
@@ -242,6 +242,22 @@ export default class Collection extends ValidationElement {
       )
     }
 
+    // Create the 'byline' which is used to separate items as well as the bottom toggle action
+    const bylineBottom = (item, index) => {
+      if (this.state.items.length < 2) {
+        return ''
+      }
+
+      return (
+        <div className="byline bottom">
+          <a href="javascript:;;" className="toggle" onClick={this.toggle.bind(this, index)}>
```

Comment:
Is this `href` to satisfy accessibility scanners?  I ask because I've run into the issue before where HTML Code Sniffer complains that a `href` is empty or points to something that doesn't exist (e.g., dynamically routed stuff).